src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/resource/kubernetes/custom/NamespacedCustomResourcesProvider.kt

```diff
@@ -18,6 +18,7 @@ import io.fabric8.kubernetes.client.dsl.Watchable
 import io.fabric8.kubernetes.client.dsl.base.CustomResourceDefinitionContext
 import org.jboss.tools.intellij.kubernetes.model.resource.NamespacedResourcesProvider
 import org.jboss.tools.intellij.kubernetes.model.resource.ResourceKind
+import org.jboss.tools.intellij.kubernetes.model.util.createContext
 import org.jboss.tools.intellij.kubernetes.model.util.getVersion
```

Comment:
idk why IJ doesn't remove it automatically but as the `GetVersion` is not used anymore this import can be removed.